### PR TITLE
[Sensitivity notebook] Add documentation

### PR DESCRIPTION
**Pull request by colinepiloquet**
**Please check if the PR fulfills these requirements**

- [x] The commit message follows our guidelines

**Does this PR already have an issue describing the problem?**

No

**What kind of change does this PR introduce?**

Add some explanations in the sensitivity notebook

**What is the current behavior?**

Just a copy/paste from a small part of another notebook.

**What is the new behavior (if this is a feature change)?**
It looks a bit more like a real independent notebook :) 

**Does this PR introduce a breaking change or deprecate an API?**
- [ ] Yes
- [x] No

### CODE REVIEW

File: sensitivity_analysis.ipynb [hunk 1]
```diff
@@ -112,64 +993,205 @@
...
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can examine the shift keys for the zone representing France. They're linked to the active power setpoint of the generators:"
+   ]
+  },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 37,
```
Comment (So-Fras) [inline sensitivity_analysis.ipynb]:
```suggestion
   "execution_count": null,
```

File: sensitivity_analysis.ipynb [hunk 2]
```diff
@@ -112,64 +993,205 @@
...
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{'SO_G1': 480.0, 'SO_G2': 480.0, 'N_G': 0.0}"
+      ]
+     },
+     "execution_count": 37,
```
Comment (So-Fras) [inline sensitivity_analysis.ipynb]:
```suggestion
     "execution_count": null,
```

File: sensitivity_analysis.ipynb [hunk 3]
```diff
@@ -112,64 +993,205 @@
...
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's create the analysis and set the zones to the two we previously declared."
+   ]
+  },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 38,
```
Comment (So-Fras) [inline sensitivity_analysis.ipynb]:
```suggestion
   "execution_count": null,
```

File: sensitivity_analysis.ipynb [hunk 4]
```diff
@@ -178,46 +1200,103 @@
...
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "And we can create the factor matrix again. This time, we're looking at how changing the net position of the two zones, `FR` and `IT`, affects the active power flow of the lines `S_SE_1` and `S_SE_2`."
+   ]
+  },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 39,
```
Comment (So-Fras) [inline sensitivity_analysis.ipynb]:
```suggestion
   "execution_count": null,
```

File: sensitivity_analysis.ipynb [hunk 5]
```diff
@@ -178,46 +1200,103 @@
...
    "source": [
-    "ptdf_results = sa.run(n6b, params)\n",
-    "m1 = ptdf_results.get_branch_flows_sensitivity_matrix()"
+    "We can run the analysis and get the sensitivity matrix."
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 40,
```
Comment (So-Fras) [inline sensitivity_analysis.ipynb]:
```suggestion
   "execution_count": null,
```

File: sensitivity_analysis.ipynb [hunk 6]
```diff
@@ -178,46 +1200,103 @@
...
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "    S_SE_1  S_SE_2\n",
+       "FR     0.2     0.2\n",
+       "IT    -0.1    -0.1"
+      ]
+     },
+     "execution_count": 40,
```
Comment (So-Fras) [inline sensitivity_analysis.ipynb]:
```suggestion
     "execution_count": null,
```

File: sensitivity_analysis.ipynb [hunk 7]
```diff
@@ -178,46 +1200,103 @@
...
+    "Transferring 1 MW of active power from the zone `FR` to the zone `IT` will cause a 0.3 MW shift on border line `S_SE_1`.\n",
     "\n",
-    "Let’s obtain that directly. After a sensitivity analysis where we should set the zones, we are able to ask for a FR zone to slack sensitivity, a FR to IT zone to zone sensitivity, a IT to FR zone to zone sensitivity and a IT zone to slack sensitivity, on all the border lines ‘S_SE_1’, ‘S_SE_2’, ‘SE_NE_1’ and ‘SE_NE_2’."
+    "Let’s obtain that directly. After running a sensitivity analysis to figure out the best zones, we can request a `FR` zone to slack sensitivity, a `FR` to `IT` zone to zone sensitivity, an `IT` to `FR` zone to zone sensitivity, and an `IT` zone to slack sensitivity on all the border lines `S_SE_1`, `S_SE_2`, `SE_NE_1` and `SE_NE_2`."
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 41,
```
Comment (So-Fras) [inline sensitivity_analysis.ipynb]:
```suggestion
   "execution_count": null,
```

File: sensitivity_analysis.ipynb [hunk 8]
```diff
@@ -229,9 +1308,82 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 42,
```
Comment (So-Fras) [inline sensitivity_analysis.ipynb]:
```suggestion
   "execution_count": null,
```

File: sensitivity_analysis.ipynb [hunk 9]
```diff
@@ -229,9 +1308,82 @@
...
+      ],
+      "text/plain": [
+       "          S_SE_1  S_SE_2  SE_NE_1  SE_NE_2\n",
+       "FR           0.2     0.2      0.2      0.2\n",
+       "FR -> IT     0.3     0.3     -0.2     -0.2\n",
+       "IT -> FR    -0.3    -0.3      0.2      0.2\n",
+       "IT          -0.1    -0.1      0.4      0.4"
+      ]
+     },
+     "execution_count": 42,
```
Comment (So-Fras) [inline sensitivity_analysis.ipynb]:
```suggestion
     "execution_count": null,
```

File: sensitivity_analysis.ipynb [hunk 10]
```diff
@@ -11,30 +11,73 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "This notebook illustrates how to run a sensitivity analysis.\n"
+    "In this notebook, you will learn how to run a sensitivity analysis on a network using pyPowsybl.\n",
```
[the remark this answers is not in the document]
Comment (colinepiloquet) [reply sensitivity_analysis.ipynb]: done

File: sensitivity_analysis.ipynb [hunk 11]
```diff
@@ -11,30 +11,73 @@
...
-   "execution_count": null,
+   "execution_count": 24,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Defaulting to user installation because normal site-packages is not writeable\n",
```
[the remark this answers is not in the document]
Comment (colinepiloquet) [reply sensitivity_analysis.ipynb]: also done

File: sensitivity_analysis.ipynb [hunk 12]
```diff
@@ -11,30 +11,73 @@
...
+    }
+   ],
    "source": [
     "pip install pypowsybl"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 25,
```
[the remark this answers is not in the document]
Comment (colinepiloquet) [reply sensitivity_analysis.ipynb]: Null means that the cell has not been executed, then each cell gets the number associated to the order in which it was executed. As we want to keep the output displayed, "execution_count" cannot be null.

**Pull request merged**